bin/install_uninstall.ml

```diff
@@ -379,6 +379,14 @@ let install_uninstall ~what =
             ~doc:
               "Make the binaries relocatable (the installation directory can \
                be moved).")
+    and+ create_install_files =
```

Comment:
If it's not actually installing, why does it need to be under the `install` subcommand? Can't this be done just by building a target.